pkgs/tools/security/notary/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchurl }:
+
+let
+    linuxPredicate = stdenv.system == "x86_64-linux";
+    darwinPredicate = stdenv.system == "x86_64-darwin";
+    metadata = assert linuxPredicate || darwinPredicate;
+        if linuxPredicate then
+            { arch = "Linux-amd64";
+              sha256 = "06cd02c4c2e7a3b1ad9899b03b3d4dde5392d964c675247d32f604a24661f839"; }
+        else
+            { arch = "Darwin-amd64";
+              sha256 = "1e32aad049d2835b0c5b4805f9abc39b88c4de90b9743775c33c9faaf6c6c383"; };
+in stdenv.mkDerivation rec {
+    shortname = "notary";
+    name = "${shortname}-${version}";
+    version = "0.4.3";
+
+    src = fetchurl {
+        url = "https://github.com/theupdateframework/notary/releases/download/v${version}/${shortname}-${metadata.arch}";
+        sha256 = metadata.sha256;
+    };
```

Comment:
Any reason not to compile from source? The build looks straight forward:
https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=notary-git

Reply:
That was to go as quick as I needed.. I can take a look at compiling for source :wink: